partiql-lang/src/main/kotlin/org/partiql/lang/compiler/PartiQLCompilerPipeline.kt

```diff
@@ -56,7 +56,7 @@ class PartiQLCompilerPipeline(
         @JvmStatic
         fun standard() = PartiQLCompilerPipeline(
             parser = PartiQLParserBuilder.standard().build(),
-            planner = PartiQLPlannerBuilder.standard().build(),
+            planner = PartiQLPhysicalPlannerBuilder.standard().build(),
```

Comment:
I'm curious why this is all being renamed to PartiQLPhysicalPlannerBuilder rather than keeping as is. Recall that the only differences between the logical and physical domains is the "operation" key which means the "physical" planning responsibilities are almost nothing. I had envisioned the planner is purely logical, and the compiler is responsible for adding the operators. 

We would need to define how one might decorate a logical plan with some name, but the physical plan plays no substantial role to warrant a big split.

Reply:
Given that we're only trying to move out the planner (returning a logical plan) to a new subproject (with `PartiQLPlanner` being its name), I decided to adjust `:partiql-lang`'s planner (returning a physical plan) to `PartiQLPhysicalPlanner`. My thinking was that, eventually, we'll want to decide on where the physical planning/compiling components should be moved, but for now, we really just need to strip out the logical plan components and make it *slightly* easier to tell which component returns a logical plan vs a physical plan.

This way, consumers of the experimental `PartiQLCompilerPipeline` can still use the API almost as-is, and we can begin modifying the `PartiQLPlanner` to return the Sprout-generated logical plan.